connectors/sync_job_runner.py

```diff
@@ -298,7 +298,7 @@ async def sync_starts(self):
             )
 
         try:
-            await self.connector.sync_starts()
+            await self.connector.sync_starts(job_type=job_type)
```

Comment:
Should `job_type` become mandatory instead of being a named parameter with a default?